paddle/pten/api/lib/utils/storage.h

```diff
@@ -75,6 +75,24 @@ class SharedStorage : public pten::Storage {
     return allocation_;
   }
 
+  // Temporary method: For compatible with fluid Tensor and improve performance
+  void ResetAllocation(std::shared_ptr<paddle::memory::Allocation> allocation,
+                       size_t offset) {
+    allocation_ = allocation;
+    data_ = pten::Allocation(
```

Comment:
Maybe cause value error, if we resize tensor in kernel. Anyway, we can solve it in the future

Reply:
I know this point, will change kernel output share rule in next pr